### PR TITLE
Configurable touch trail

### PR DESCRIPTION
**Pull request by DevamBhatia216**

### CODE REVIEW

File: 8vim/lint-report.html [hunk 1]
```diff
@@ -0,0 +1,1339 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
```
Comment (flide) [inline 8vim/lint-report.html]: No need to commit this file.

File: 8vim/src/main/java/inc/flide/vim8/ui/LauncherActivity.java [hunk 2]
```diff
@@ -1,23 +1,33 @@
 package inc.flide.vim8.ui;
 
+import android.annotation.SuppressLint;
```
Comment (flide) [inline 8vim/src/main/java/inc/flide/vim8/ui/LauncherActivity.java:3]: You should not use SuppressLint, if there is an issue with lint, try to find the root cause.

File: 8vim/src/main/java/inc/flide/vim8/ui/LauncherActivity.java [hunk 3]
```diff
@@ -45,6 +55,9 @@
     private Button yellow_button;
     private Button blue_button;
 
+    private ConstraintLayout constraintLayout_select_color;
+
+    @SuppressLint("ClickableViewAccessibility")
```
Comment (flide) [inline 8vim/src/main/java/inc/flide/vim8/ui/LauncherActivity.java]: This particular suppressLint is allowed by the way, will add it to the lint config, so rule sticks around. Don't use SuppressLint annotation.

File: 8vim/src/main/java/inc/flide/vim8/ui/LauncherActivity.java [hunk 4]
```diff
@@ -84,51 +99,125 @@ protected void onCreate(Bundle savedInstanceState) {
...
+        else{
+            constraintLayout_select_color.setVisibility(View.INVISIBLE);
+        }
+
+
 
         red_button = (Button) findViewById(R.id.red_button);
-        red_button.setOnClickListener(new View.OnClickListener() {
+        red_button.setOnTouchListener(new View.OnTouchListener() {
+            @SuppressLint("ClickableViewAccessibility")
```
Comment (flide) [inline 8vim/src/main/java/inc/flide/vim8/ui/LauncherActivity.java]: don't use

**Review by flide — CHANGES_REQUESTED** [review]
Fix lint issues, don't suppress them, don't ignore them.

File: 8vim/src/main/java/inc/flide/vim8/ui/LauncherActivity.java (hunk 2 above)
Comment (DevamBhatia216) [reply 8vim/src/main/java/inc/flide/vim8/ui/LauncherActivity.java:3]: yeah understood bro

**Review by flide — APPROVED** [review]

**Pull request merged**